lib/bolt/cli.rb

```diff
@@ -611,6 +604,18 @@ def migrate_project
       ok ? 0 : 1
     end
 
+    # Collect any values with the matching key
+    # This will be needed to properly migrate inventory files since v2 files
+    # no longer require a 'version' key
+    def collect_vals(data, key)
+      case data
+      when Array
+        data.flat_map { |d| collect_vals(d, key) }
+      when Hash
+        [data['nodes']] + collect_vals(data.values, key).compact
+      end
+    end
```

Comment:
Since v1 files more than likely won't have a `version` field set, and since v2 files are becoming the default, `migrate_project` can't rely on checking the `version` field for migrating. The best solution I could come up with was checking if any `nodes` keys are present and then migrating if there are.

I'm positive there's a better way to check a nested hash for a key than this, but this is the best I could come up with.